<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Báo cáo doanh thu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1101</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	background-color:rgb(255, 255, 127);
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
	background-color:rgb(0, 85, 255);
	border-radius: 15px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Báo cáo doanh thu&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnInbaocao">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>570</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size: 20px;
    border-radius: 10px;
	background-color: rgb(0, 85, 255);
	color:rgb(255, 255, 255);
	font-weight: bold;
}
QPushButton:hover{
	background-color: rgb(0, 170, 255);
	cursor: pointer;
}</string>
     </property>
     <property name="text">
      <string>In báo cáo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnThoatb">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>570</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size: 20px;
    border-radius: 10px;
	background-color: rgb(0, 85, 255);
	color:rgb(255, 255, 255);
	font-weight: bold;
}
QPushButton:hover{
	background-color: rgb(0, 170, 255);
	cursor: pointer;
}</string>
     </property>
     <property name="text">
      <string>Thoát</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>1001</width>
       <height>431</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Số lượng vé bán ra</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Số lượng dịch vụ</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
